dronegen/push.go

```diff
@@ -1,6 +1,175 @@
 package main
 
+import "fmt"
+
 func pushPipelines() []pipeline {
-	// TODO: migrate
-	return nil
+	return []pipeline{
+		pushPipeline(buildType{os: "linux", arch: "amd64"}),
+		pushPipeline(buildType{os: "linux", arch: "i386"}),
+		pushPipeline(buildType{os: "linux", arch: "amd64", fips: true}),
+		pushPipeline(buildType{os: "windows", arch: "amd64"}),
+		pushPipeline(buildType{os: "linux", arch: "arm"}),
+		pushPipeline(buildType{os: "linux", arch: "arm", fips: true}),
+		pushPipeline(buildType{os: "linux", arch: "arm64"}),
+		pushPipeline(buildType{os: "linux", arch: "arm64", fips: true}),
+	}
+}
+
+// translatePipelineArch converts a nominal arch to a Go architecture representation
+// e.g. i386 -> 386, x86_64 -> amd64
+// This is required because the existing pipeline uses i386 whereas `GOARCH` is expected to be 386
+func translatePipelineArch(arch string) string {
+	if arch == "i386" {
+		return "386"
+	}
+	return arch
+}
+
+func pushBuildCommands(params buildType) []string {
+	commands := []string{
+		`apk add --no-cache make`,
+		`chown -R $UID:$GID /go`,
+	}
```

Comment:
nit: move this further down, once `makeCommand` is set
that way you'll need less appending.